src/window/basic_frame.rs

```diff
@@ -390,21 +402,21 @@ impl Frame for BasicFrame {
             // draw the grey background
             {
                 let mut writer = BufWriter::new(&mut *pool);
-                // For every pixel in top border
-                for y in 0..DECORATION_TOP_SIZE {
-                    for _ in 0..DECORATION_SIZE {
+                // For every pixel in header
+                for y in 0..HEADER_SIZE {
+                    for _ in 0..BORDER_SIZE {
                         let _ = writer.write_u32::<NativeEndian>(0x00_00_00_00);
                     }
```

Comment:
I think the lateral transparent drawings on the header are redundant with the lateral subsurfaces, no?

Reply:
Unless I understood you wrong or I didn't test properly, the transparent color of the borders won't overwrite the color of the headers when layered on top of each other.

Reply:
Actually I can implement the positioning and change the width of the header so the drawing of transparent color is redundant

Reply:
I just meant that the header subsurface does not need to be `HEADER_SIZE + 2*BORDER_SIZE` large, but only `HEADER_SIZE`. Isn't that the case?

Reply:
Yeah and also the positioning needs to change from
```
 (-(BORDER_SIZE as i32), -(HEADER_SIZE as i32)
```
to
```
(0, -(HEADER_SIZE as i32)
```
otherwise it renders color to the left of the window.

Reply:
Does the resize and position headerbar commit do what you were thinking of